src/components/Subscriptions/SubscriptionsMonthlyGood.vue

```diff
@@ -255,10 +267,37 @@ export default {
 				this.mgAmount = autoDepositAmount - this.donation;
 				this.paymentMethod = data?.my?.autoDeposit?.paymentMethod ?? {};
 				this.subscriptionId = data?.my?.autoDeposit?.id;
+				this.firstName = data?.my?.userAccount?.firstName ?? '';
+				this.subscriptionsLoans = data?.mySubscriptions?.values?.length ?? 0;
+				const mySubscriptions = data?.mySubscriptions?.values?.history?.values ?? [];
```

Comment:
This doesn't work at all. It should be: `data?.mySubscriptions?.values[0]?.history?.values` because data?.mySubscriptions?.values is an array